src/adapters/scene.js

```diff
@@ -83,6 +83,15 @@ Scene.prototype.initMapBox = async function(locationHash) {
 
   this.popup.init(this.mb);
 
+  // iframe: no interactions
+  if (window.no_ui) {
+    this.mb['scrollZoom'].disable();
+    this.mb['dragPan'].disable();
+    this.mb['doubleClickZoom'].disable();
+    this.mb['dragRotate'].disable();
+    this.mb['touchZoomRotate'].disable();
+  }
```

Comment:
Not tested, but it could be cleaner to simply add `interactive: false` in the `Map` options:
https://docs.mapbox.com/mapbox-gl-js/api/map/#map-parameters